Libplanet/Net/Messages/GetRecentStates.cs

```diff
@@ -47,5 +48,17 @@ protected override IEnumerable<NetMQFrame> DataFrames
                 }
             }
         }
+
+        [Serializable]
+        internal class InvalidTargetException : SwarmException
```

Comment:
I guess we need a protected constructor for [`Serializable`].

Reply:
Added a constructor for serialization.